roles/lib_openshift/src/doc/user

```diff
@@ -0,0 +1,66 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_user
+short_description: Create, modify, and idempotently manage openshift users.
+description:
+  - Modify openshift users programmatically.
+options:
+  state:
+    description:
+    - State controls the action that will be taken with resource
+    - 'present' will create or update a user to the desired state
+    - 'absent' will ensure user is removed
+    - 'list' will read and return a list of users
+    default: present
+    choices: ["present", "absent", "list"]
+    aliases: []
+  kubeconfig:
+    description:
+    - The path for the kubeconfig file to use for authentication
+    required: false
+    default: /etc/origin/master/admin.kubeconfig
+    aliases: []
+  debug:
+    description:
+    - Turn on debug output.
+    required: false
+    default: False
+    aliases: []
+  username:
+    description:
+    - Short username to query/modify.
+    required: false
+    default: None
+    aliases: []
+  full_name:
+    description:
+    - String with the full name/description of th user.
+    required: false
+    default: None
+    aliases: []
+  groups:
+    description:
+    - List of groups the user should be a member of.
+    required: false
+    default: []
+    aliases: []
+author:
+- "Joel Diaz <jdiaz@redhat.com>"
+extends_documentation_fragment: []
+'''
+
+EXAMPLES = '''
+- name: Ensure user exists
+  oc_user:
+    state: present
+    username: johndoe
+    full_name "John Doe"
+
+- name: Ensure user does not exist
+  oc_user:
+    state: absent
+    username: johndoe
```

Comment:
Another thing that would be helpful in the docs is to document return values, since it has been a source of confusion in some of the PRs that are migrating to using the modules.